modules/ROOT/pages/create-instance-task.adoc

```diff
@@ -3,50 +3,58 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In API Manager 2.x, you can use instances of an API in multiple environments. Additionally, you can have multiple instances in the same environment. 
+In API Manager 2.x, instances of an API can exist in multiple environments. Additionally, you can create multiple instances of an API in the same environment. 
 
-You can create one instance of an API that serves as a proxy. You can create another to manage directly as a basic endpoint. You can apply caching policies (for example, to the proxy), and throttling and security policies to the basic endpoint.
+An example: 
 
-After setting up an API version for management, you repeat the process to create another instance. Attempting to set up the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
+* Create an instance of an API and configure it as a proxy. Apply caching policies to this proxy. 
```

Comment:
Make the second sentence a second action (2.), or follow the bullet list format shown in our style reference, which inserts a black line between the list item and any further detail for it....
ACTUALLY...
...looking at the rest of this, I suggest making this all one step, based on the overall task...:

Reply:
```suggestion
. Create an instance of an API, configure it as a proxy, and apply caching policies to this proxy. 
```